trashtalk/templates/users/_new.html

```diff
@@ -0,0 +1,58 @@
+<div id="signupModal" class="modal fade" role="dialog">
```

Comment:
As with the previous comment, this should be `_signup_form.html` or `_signup_modal.html`